### PR TITLE
feat(grafana): add time interval to panel titles

### LINKED ISSUE #952: Add descriptive unit to axes (interval) in Grafana
## Description

It would be nice to show the interval for aggregated statistics such as "Blocks" in the axis. For this we should create a custom unit and read out the `aggregation_interval` variable.

### PR DESCRIPTION
**Pull request by grtlr**
## Linked Issues

* Closes #952.

## Notes to Reviewer

Unfortunately, it does not seem possible to add the units to the axis, which is why I added them to the panel title.

### CODE REVIEW

File: docker/assets/grafana/dashboards/analytics_dashboard.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
         "type": "influxdb",
         "uid": "PE5723DBC504634E6"
       },
+      "description": "",
```
Comment (Alex6323) [inline docker/assets/grafana/dashboards/analytics_dashboard.json:46]: just curious...this necessary?

Comment (grtlr) [reply docker/assets/grafana/dashboards/analytics_dashboard.json:46]: Probably not. Grafana does some weirdness to the JSON depending on where you click in the GUI. I can remove it.

```suggestion
```

File: docker/assets/grafana/dashboards/analytics_dashboard.json [hunk 2]
```diff
@@ -219,14 +219,15 @@
...
-      "title": "Blocks",
+      "title": "Blocks/${aggregation_interval}",
       "type": "timeseries"
     },
     {
       "datasource": {
         "type": "influxdb",
         "uid": "PE5723DBC504634E6"
       },
+      "description": "",
```
Comment (grtlr) [inline docker/assets/grafana/dashboards/analytics_dashboard.json:230]:
```suggestion
```

File: docker/assets/grafana/dashboards/analytics_dashboard.json [hunk 3]
```diff
@@ -1432,7 +1434,8 @@
             "mode": "absolute",
             "steps": [
               {
-                "color": "green"
+                "color": "green",
+                "value": null
```
Comment (grtlr) [inline docker/assets/grafana/dashboards/analytics_dashboard.json:1438]:
```suggestion
```

File: docker/assets/grafana/dashboards/analytics_dashboard.json [hunk 4]
```diff
@@ -1432,7 +1434,8 @@
             "mode": "absolute",
             "steps": [
               {
-                "color": "green"
+                "color": "green",
```
Comment (grtlr) [inline docker/assets/grafana/dashboards/analytics_dashboard.json:1437]:
```suggestion
                "color": "green"
```

File: docker/assets/grafana/dashboards/analytics_dashboard.json [hunk 5]
```diff
@@ -1249,7 +1249,8 @@
...
+                "value": null
```
Comment (grtlr) [inline docker/assets/grafana/dashboards/analytics_dashboard.json:1253]:
```suggestion
                "color": "green"
```

**Review by Alex6323 — APPROVED** [review]

**Pull request merged**